core/objects.go

```diff
@@ -239,6 +264,21 @@ type Challenge struct {
 	AccountKey *jose.JsonWebKey `json:"accountKey,omitempty"`
 }
 
+// UnsafeSetToken sets the token value both in the Token field and in the
+// serialized AuthorizedKey object.
+//
+// This method should only be used for writing tests, in cases where the
```

Comment:
We could put it in test and make it UnsafeSetToken(ch *core.Challenge, token string)?